### ISSUE TITLE
The "double backtick problem"

### ISSUE THREAD

**Issue by jennybc** [rstudio/r-manuals#7]
Example `.texi` input:

```
... which can be found in the @file{src/library/@var{pkg}/man} directories ...
```

https://github.com/wch/r-source/blob/a505a59a02f445e4918777a585270faee97c42e8/doc/manual/R-exts.texi#L6606

Leads to this in the official `.html` manual:

``` html
... which can be found in the <samp>src/library/<var>pkg</var>/man</samp> directories ...
```

But like this in the `.html` from the Quarto-ized manual:

``` html
... which can be found in the <var>src/library/``pkg</var><code>/man</code> directories ...
```

which looks like this:

![Screen Shot 2022-10-27 at 11 45 48 AM](https://user-images.githubusercontent.com/599454/198373282-f8f5f7c8-3c45-4a8a-81fd-3028a138167d.png)

---

Example `.texi` input:

```
\name@{@var{name}@}
```

https://github.com/wch/r-source/blob/a505a59a02f445e4918777a585270faee97c42e8/doc/manual/R-exts.texi#L6727

Leads to this in the official `.html` manual:

``` html
<code>\name{<var>name</var>}</code>
```

But like this in the `.html` from the Quarto-ized manual:

``` html
<var>\name{``name</var><code>}</code>
```

which looks like this:
![Screen Shot 2022-10-27 at 11 45 57 AM](https://user-images.githubusercontent.com/599454/198373352-7dfb8f33-0897-4e21-aa90-13861f600079.png)

**Comment by jennybc** [comment]
It does feel like `<var></var>` within `<samp></samp>` (which the official `.html` ends up with) is semantically the more correct HTML here, so I wonder if some part of the pipeline can be tweaked to do these same conversions? But I also have no idea what I'm going when it comes to web things :sweat_smile:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/samp

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/var

**Comment by andrie** [comment]
I believe I have fixed the underlying issue with d62b91a.  Closing, but let me know if there are still similar remnants.
